docs/configuration.md

```diff
@@ -366,7 +366,7 @@ A three node cluster running on different hosts could be minimally configured as
     
     [gossip]
       port = 12000
-      seed = "node0.pilosa.com:12000"
```

Comment:
While this works as a single string value, it might be more informative to represent `seeds` as an array of one element (to indicate how one might provide additional seeds) in all of these examples:
```
seeds = ["node0.pilosa.com:12000"]
```